csv1.rb

```diff
@@ -0,0 +1,53 @@
+require 'csv'
+class CSVReader
```

Comment:
I can't use these class with other CSV files. The attributes are hard coded here.